src/main/java/com/systemmeltdown/robotlib/subsystems/Fallible.java

```diff
@@ -0,0 +1,8 @@
+package com.systemmeltdown.robotlib.subsystems;
+
+public interface Fallible {
```

Comment:
I like the interface, but I don't feel the name describes the capability. I feel "Failsafe" should be in this interface name somewhere.